Code/GraphMol/MolStandardize/Fragment.cpp

```diff
@@ -160,6 +160,9 @@ ROMol *LargestFragmentChooser::choose(const ROMol &mol) {
   BOOST_LOG(rdInfoLog) << "Running LargestFragmentChooser\n";
 
   std::vector<boost::shared_ptr<ROMol>> frags = MolOps::getMolFrags(mol);
+  if (frags.empty()) {
```

Comment:
See the comment in `normalize()`, it applies here too

Reply:
Checking `getNumAtoms()` was my first thought, but then I got paranoid about `getMolFrags` potentially swallowing atoms and returning an empty vector and went down the paranoid route. It happens.
Instead, I hadn't thought of the properties, and that's indeed a very good point.